### PR TITLE
Improve documentation - I

### PR DESCRIPTION
**Pull request by zm711**
I'll try to go folder by folder. If in the future you'd prefer me to just make a pull request when done with a bigger set of files I can, but I prefer making a small set of changes at time, so not too many files need to be reviewed. Happy to adjust though.

### CODE REVIEW

**Comment by h-mayorquin** [comment]
Thanks for your contribution!
 I am assuming this is documentation related edits only? I labeled this making this assumption, let me know if it is wrong.

In my view small PRs are better, the only thing that I would like is a more sepcific PR title (something like documentation correction would be useful).

**Comment by zm711** [comment]
Sorry yeah it is just documentation. Totally spaced on the titled. I should have linked #1380 too.

**Comment by h-mayorquin** [comment]
Great. Note that you can see how the documentation looks by clicking on the details at the bottom of the actions (where it says read the doccs build succeeded)

image

And then clicking on view docs (where the pointer is) as in the following screenshot:
image

It is interesting that I remember that in neuroconv clicking on the action took you directly to the documentation.  I wonder if @CodyCBakerPhD remembers how to set that up.

File: doc/index.rst [hunk 1]
```diff
@@ -64,7 +64,7 @@ To get started with SpikeInterface, you can take a look at the following additio
 
 - | The `2020 eLife paper <https://elifesciences.org/articles/61834>`_ introduces the concept and motivation and 
   | performs an in-depth comparison of multiple sorters (spolier alert: they strongly disagree between each other!). 
-  | **Note**: the code-base and implementation have changed a lot since the "paper" version in published in 2020. 
+  | **Note**: the code-base and implementation have changed a lot since the "paper" version published in 2020.
```
Comment (h-mayorquin) [inline doc/index.rst:67]: I would probably accept this without any other step but here I would probably like to know whether @alejoe91 and @samuelgarcia want to say any further words on this.

Comment (zm711) [reply doc/index.rst:67]: Yeah it possible they wanted a link to the eLife paper there? But as is it is just a floating "in". In the future I could definitely double check on those. Thanks!

**Comment by CodyCBakerPhD** [comment]
> It is interesting that I remember that in neuroconv clicking on the action took you directly to the documentation. I wonder if @CodyCBakerPhD remembers how to set that up.

It used to do that, but even on NeuroConv it no longer does. So that's probably something they changed upstream

**Comment by h-mayorquin** [comment]
I see, thanks for the update @CodyCBakerPhD.

**Comment by alejoe91** [comment]
Thanks @zm711 

Good to merge on my side :)

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**